README.md

```diff
@@ -25,17 +25,22 @@ jobs:
         uses: bump-sh/github-action@0.2
         with:
           id: <BUMP_DOC_ID>
-          token: <BUMP_DOC_TOKEN>
+          token: ${{secrets.BUMP_TOKEN}}
           file: doc/api-documentation.yml
 ```
 
 Important: [actions/checkout](https://github.com/actions/checkout) has to be called **before this action**.
 
+
 ## Inputs
 
 * `id` (required): Documentation id. Can be found in the documentation settings on https://bump.sh
```

Comment:
shouldn't we update `id` -> `doc` here too?

Reply:
its in the 2nd commit.